lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -354,6 +358,18 @@ FUNCTIONKIND_VALUES(VALUE)
         case FunctionKind::Array_indexOf:
             library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryIndexOf, func);
             break;
+        case FunctionKind::Array_some:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntrySome, func);
+            break;
+        case FunctionKind::Array_every:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryEvery, func);
+            break;
+        case FunctionKind::Array_includes:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryIncludes, func);
+            break;
+        case FunctionKind::Array_reduce:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryReduce, func);
+            break;
         }
```

Comment:
nit: I was hoping to have default case as well (which should not be reached and do Assertorfailfast).

Reply:
7a249b412b8679e7157949b794aedc7b19fc7252